lib/tasks/publishing_api/publish_api_prefix.rake

```diff
@@ -0,0 +1,33 @@
+require 'logger'
+require 'gds_api/publishing_api'
+require 'gds_api/publishing_api/special_route_publisher'
+
+namespace :publishing_api do
+  desc "Publish /api route to publishing_api"
+  task publish_api_prefix: :environment do
+    puts "Content Tagger is claiming path /api"
+    endpoint = Services.publishing_api.options[:endpoint_url]
+    Services.publishing_api.put_json("#{endpoint}/paths/api", publishing_app: "content-tagger", override_existing: true)
+
+    publishing_api = GdsApi::PublishingApiV2.new(
+      Plek.new.find('publishing-api'),
+      bearer_token: ENV['PUBLISHING_API_BEARER_TOKEN'] || 'example'
+    )
+
+    special_route_publisher = GdsApi::PublishingApi::SpecialRoutePublisher.new(
+      logger: Logger.new(STDOUT),
+      publishing_api: publishing_api
+    )
+
+    puts "Publishing /api..."
+    special_route_publisher.publish(
+      title: 'Public content API',
+      description: 'The unsupported GOV.UK content API (https://insidegovuk.blog.gov.uk/2014/09/15/current-state-of-apis-on-gov-uk/)',
```

Comment:
Can we make it clear in the description what it's used for (search / content-store / others?). That lessens the chance that we remove it again.